### PR TITLE
+diff-so-fancy

### PR DESCRIPTION
**Pull request by davdroman**
Based off https://github.com/Homebrew/homebrew-core/blob/480317f32d9a2908002451092ac49b0d001541cd/Formula/diff-so-fancy.rb, but I need some help to make it work 🤔

### CODE REVIEW

File: projects/github.com/so-fancy/diff-so-fancy/package.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+versions:
+  github: so-fancy/diff-so-fancy
+  strip: /^v/
+
+build:
+  script: |
+    chmod +x diff-so-fancy
+    mkdir -p "{{prefix}}/bin"
+    ln -sf "$PWD/diff-so-fancy" "{{prefix}}/bin"
```
Comment (davdroman) [inline projects/github.com/so-fancy/diff-so-fancy/package.yml]: Something's definitely wrong with this but I'm not sure what it is.

Comment (jhheider) [reply projects/github.com/so-fancy/diff-so-fancy/package.yml]: That's hardcoding the build $PWD in the symlink. You want `mv diff-so-fancy "{{prefix}}/bin/"`.

**Comment by jhheider** [comment]
there you go! :)

**Comment by davdroman** [comment]
@jhheider ah I see! I’d have never figured that out myself. Thanks for the help! :)

**Pull request merged**